server/seed/demoSeed/generateCohorts.js

```diff
@@ -10,6 +10,11 @@ const generateCohorts = (demoUserId, userIds) => [
     description:
       'You are the owner of this cohort. You can create sacks, add members and manage their permissions, modify and archive cohort.',
     isArchived: false,
+    isDeleted: false,
```

Comment:
Consider using constructors so default parameters can be assigned automatically:

```javascript
const Cohort = ({ description = '', isArchived = false, locks = { description = false, name = false } = {}, ... }) => ({
  _id: ObjectId(),
  description,
  isArchived,
  locks,
  ...
});

const generateCohorts = (demoUserId, userIds) => [
  Cohort({ description: 'You are the owner of this cohort. You can create sacks, add members and manage their permissions, modify and archive cohort.' }),
  ...
]
```